### PR TITLE
chore(QTM-408): Fixed styled-components version

### PR DESCRIPTION
**Pull request by MarcosViniciusPC**
## Description
https://jirasoftware.catho.com.br/browse/QTM-408

## Review guide
- [ ] Unit tests (yarn test:components)
- [ ] Code review
- [ ] renders in SSR simulator

### CODE REVIEW

File: components/Dropdown/Dropdown.jsx [hunk 1]
```diff
@@ -169,8 +169,8 @@ DropContainer.displayName = 'DropContainer';
 
 const _getValue = item => (item ? item.value || item.label || item : '');
 const _getLabel = item => (item ? item.label || item.value || item : '');
-const _getImage = item => (item ? item.img : null);
-const _getImageAlt = item => (item ? item.alt : null);
+const _getImage = ({ img }) => img;
+const _getImageAlt = ({ alt }) => alt;
```
Comment (toncabral88) [inline components/Dropdown/Dropdown.jsx:173]: I didn't get this. What has changed here?

**Review by toncabral88 — APPROVED** [review]
[x] Code review
[x] Unit

File: components/Dropdown/Dropdown.jsx (hunk 1 above)
Comment (MarcosViniciusPC) [reply components/Dropdown/Dropdown.jsx:173]: There was an update to eslint that added new rules that broke the previous implementation in this code snippet

**Review by rapahaeru — APPROVED** [review]
## Review guide
- [x] Unit tests (yarn test:components)
- [x] Code review
- [x] renders in SSR simulator

**Pull request merged**